Tools/cases_generator/generate_cases.py

```diff
@@ -540,6 +540,18 @@ def write_metadata(self, metadata_filename: str, pymetadata_filename: str) -> No
                 for name in self.opmap:
                     self.out.emit(f'[{name}] = "{name}",')
 
+            with self.metadata_item(
+                f"const uint8_t _PyOpcode_Caches[256]",
+                "=",
+                ";",
+            ):
+                for name, family in self.families.items():
+                    instr = self.instrs[name]
+                    if instr.cache_offset > 0:
+                        self.out.emit(f'[{name}] = {instr.cache_offset},')
+                # Irregular case:
+                self.out.emit('[JUMP_BACKWARD] = 1,')
```

Comment:
How should we deal with this case?

Reply:
Put JUMP_BACKWARD in a family of its own, and add the cache effect notation to its `instr`? Or is that a problem because it's also part of the JUMP pseudo?

If it's complicated I'm fine with leaving this special case.

Reply:
Alas, it looks like it's in a pseudo with JUMP_FORWARD and those are required to have the same size. Maybe that requirement is too strict?

Reply:
In an offline discussion with @markshannon we decided to add a family for `JUMP_BACKWARD` which would declare its cache size. This causes the generator to emit an assertion that the cache size declared for the family (1) is equal to that which is calculated from the instruction signature (0).  If I add the cache to the signature, then I get an error about the pseudo-instructions for `JUMP` (`JUMP_FORWARD` and `JUMP_BACKWARD`) not having the same instruction format.

So there is something here that we need to resolve, and it's not a trivial change. I suggest we commit this PR (which is large enough) with the workaround on line 553, and resolve it in a separate PR.

Reply:
Agreed. LGTM.